stubs/TYPO3/CMS/Frontend/Controller/TypoScriptFrontendController.php

```diff
@@ -330,4 +332,16 @@ public function addTempContentHttpHeaders()
     public function sendCacheHeaders()
     {
     }
+
+    /**
+     * @return void
+     */
+    public function checkEnableFields($row, $bypassGroupCheck = false)
```

Comment:
why is `void` not added as native return type here?

Reply:
i checked some other stubs and was wondering on the void handling, sometimes its just an annotation, sometimes its done with proper return turn
And besides that the void is not the original one in most cases (would guess thats because of phpstan? and some other stuff?)

Reply:
@helsner Use what it´s used at the moment in the core class itself. If it a docblock use the docblock.

Reply:
ok.